tests/test_service.py

```diff
@@ -123,7 +123,26 @@ def test_get_wishlist_not_found(self):
         resp = self.app.get("/wishlists/0")
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
 
-   
+	def test_create_wishlist_with_missing_args(self):
```

Comment:
@yf1357 - There is a tab error here, the master branch is breaking. Please check.